### PR TITLE
Update Status.tsx

### PR DESCRIPTION
**Pull request by AAkashyap1**

### CODE REVIEW

File: src/components/ProblemsPage/Status.tsx [hunk 1]
```diff
@@ -4,10 +4,12 @@ import UserDataContext from '../../context/UserDataContext/UserDataContext';
 import { useDarkMode } from '../../context/DarkModeContext';
 
 const options = [
+  { label: 'Not Attempted', value: 'Not Attempted' },
```
Comment (maggie-j-liu) [inline src/components/ProblemsPage/Status.tsx:7]: if the labels and values are all the same maybe import from here https://github.com/cpinitiative/usaco-guide/blob/master/src/models/problem.ts#L806?

**Comment by nathangong** [comment]
Could you also fix the problem where the background abruptly ends if there aren't that any problems that show up?

image

File: src/components/ProblemsPage/Status.tsx (hunk 1 above)
Comment (AAkashyap1) [reply src/components/ProblemsPage/Status.tsx:7]: Good idea but i think react-select has some weird requirements where all items need to have a label (for option display) and a value

Comment (AAkashyap1) [reply src/components/ProblemsPage/Status.tsx:7]: idk how to combat that

**Comment by AAkashyap1** [comment]
> Could you also fix the problem where the background abruptly ends if there aren't that many problems that show up?
> 
> image

i think i fixed this - can you check it when the deployment finishes?

**Comment by nathangong** [comment]
Yep, it looks good

**Pull request merged**